src/Ombi/ClientApp/src/app/discover/components/genre/genre-button-select.component.scss

```
@@ -1,35 +1,71 @@
+        color: rgba(255, 255, 255, 0.8);
+        padding: 0;
+        transition: all 0.2s ease;
+
+        &:hover {
+            background: rgba(255, 255, 255, 0.12);
+            border-color: rgba(255, 255, 255, 0.2);
+            color: #fff;
+        }
+    }
+
+    ::ng-deep .mat-button-toggle-appearance-standard .mat-button-toggle-label-content {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)